main/Helper.cpp

```diff
@@ -1140,6 +1145,117 @@ int SetThreadName(std::thread::native_handle_type thread, const char *name)
 }
 #endif
 
+#if !defined(PTRACE_ATTACH) && defined(PT_ATTACH)
```

Comment:
OK, good point, it is not needed.
(The idea is to be compatible with both Linux and Mac OSX / BSD, but since this code is not not used for Linux it is not needed)